### PR TITLE
[Doc Improvement] [Updates teams app settings][3002790]

### PR DESCRIPTION
**Pull request by v-ganr**

### CODE REVIEW

**Comment by Danielabom** [comment]
Contacted author to get info.

File: api-reference/beta/api/teamsappsettings-update.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ PATCH /teamwork/teamsAppSettings
 
 |Property|Type|Description|
 |:---|:---|:---|
-|allowUserRequestsForAppAccess|Boolean|Indicates whether Teams users are allowed to request admins access to certain Teams apps.|
+|allowUserRequestsForAppAccess|Boolean|Indicates whether Teams users are allowed to request unavailable Teams apps.|
```
Comment (Danielabom) [inline api-reference/beta/api/teamsappsettings-update.md:53]: I am a little confused here. Are the users no longer requesting access to the app but requesting an unavailable app download?

File: api-reference/beta/resources/teamsappsettings.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Inherits from [entity](../resources/entity.md).
 ## Properties
 |Property|Type|Description|
 |:---|:---|:---|
-|allowUserRequestsForAppAccess|Boolean|Indicates whether Teams users are allowed to request admins access to certain Teams apps.|
+|allowUserRequestsForAppAccess|Boolean|Indicates whether Teams users are allowed to request unavailable Teams apps.|
```
Comment (Danielabom) [inline api-reference/beta/resources/teamsappsettings.md:29]: I am a little confused here. Are the users no longer requesting access to the app but requesting an unavailable app download?

**Review by Danielabom — CHANGES_REQUESTED** [review]
Hi @v-ganr, thank you for working on this PR. I just have a couple of questions. Thanks!

File: api-reference/beta/resources/teamsappsettings.md (hunk 2 above)
Comment (v-ganr) [reply api-reference/beta/resources/teamsappsettings.md]: Hi @Danielabom 

I have updated the description as " Indicates whether users are allowed to request access to unavailable Teams apps"

File: api-reference/beta/api/teamsappsettings-update.md (hunk 1 above)
Comment (v-ganr) [reply api-reference/beta/api/teamsappsettings-update.md]: Hi @Danielabom 

I have updated the description as " Indicates whether users are allowed to request access to unavailable Teams apps."

**Review by Danielabom — APPROVED** [review]
Thank you for working on this PR @v-ganr. PR approved.

**Pull request merged**